### ISSUE TITLE
DOTA 2 INPUT LAG

### ISSUE THREAD

**Issue by noregrets1** [ValveSoftware/Dota2-Gameplay#9919]
Sometimes Dota have input lag doesn't matter if you have 100, 200 or 300 FPS and it only happens when valve working on updates after they done and release patch game feels normally and responsive, for a day or two, or hour... Until they start doing this again.

When it happens it is 2x harder to play, you may experience muscle tension, distracting, headache, too fast fatigue, lose focus on the game etc, it can happen even after 1 or 2 match. You can think it's on you, you just need to rest or relax, but it's not.

I play this game for a really long time, and it been working like this for years.

The biggest problem about this is that it's not really noticible until you starting realize when game works normal and when it's not, and of course of self-criticism.

I don't know how to prove it yet, but just try to play and pay attention on your condition, the simpliest way to describe it if you're not satisfied by the game even after you just won a match - this is it.

I tried to chat with JeffHill but he just gave up on me and on this problem.
IT ALWAYS HAPPENING WHEN YOU STARTING UPDATE GAME COORDINATOR

**Comment by Aluerie** [comment]
Duplicate #5436

---
Please, close your duplicate issue and instead upvote the original one with "👍" reaction. Comment extra details if any.

And in future, first, use search to find out if the bug you want to report was already submitted by somebody else so you can upvote it and not clutter the repository with duplicates.

**Comment by TpsArchyy** [comment]
Yes i have the same issue, dota 2 has some wierd input lag at random times... Fps are 144+ (my fps slider is set to 145), ping is always 20 or 21 rarely above but sometimes i feel like in some games when i press the button it takes like half a seccond to a seccond for the server to respond to my command. 

As for when this happens regarding game coordinator, i have no idea if they are connected but i can confirm that the input lag issue is there but do not know what causes it.

**Comment by justone92** [comment]
Same as me. I have some weird input lag from time to time. Like I can play 1-2 games perfectly fine but next can be disaster. I have all the time around 120 fps and 25 ping, with no change, but input delay appears for some reason. Every single command have like 0.2 sec delay, its almost unplayable. It drove me crazy. I changed whole pc. I bought new motherboard, GPU, CPU, new fresh OS, new keyboard, mouse and at end monitor. Then I started to play again. A few games was like perfect. I was like finally I can play Dota, then suddenly all the same. I felt sick from my stomach...Tested internet connection, no issue, good ping, no packet loss, all drivers updated... I don't know what to do, it makes me to quit playing.

**Comment by noregrets1** [comment]
> Same as me. I have some weird input lag from time to time. Like I can play 1-2 games perfectly fine but next can be disaster. I have all the time around 120 fps and 25 ping, with no change, but input delay appears for some reason. Every single command have like 0.2 sec delay, its almost unplayable. It drove me crazy. I changed whole pc. I bought new motherboard, GPU, CPU, new fresh OS, new keyboard, mouse and at end monitor. Then I started to play again. A few games was like perfect. I was like finally I can play Dota, then suddenly all the same. I felt sick from my stomach...Tested internet connection, no issue, good ping, no packet loss, all drivers updated... I don't know what to do, it makes me to quit playing.

Its all on valve and they know about it but won't do anything. Something is fucking up DOTA it happens when they work on updates. And this "game coordinator" non stop reconecting during the game...

**Comment by IgorHoruzha** [comment]
#15559 Do You still have the issue?

**Comment by akioda101** [comment]
Had a few days of normal games. 
And now it happened again. Input lag only went away when I restarted my router, and that took 7 minutes

So now I'm banned from MM and in Low Prio... again....

Valve, fix your fk game
